app/assets/stylesheets/deb.scss

```diff
@@ -0,0 +1,108 @@
+.organization-details {
+  html, body, div, span, applet, object, iframe,
+  h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+  abbr, acronym, address, big, cite, code,
+  del, dfn, em, img, ins, kbd, q, s, samp,
+  small, strike, strong, sub, sup, tt, var,
+  b, u, i, center,
+  dl, dt, dd, ol, ul, li,
+  fieldset, form, label, legend,
+  table, caption, tbody, tfoot, thead, tr, th, td,
+  article, aside, canvas, details, embed,
+  figure, figcaption, footer, header, hgroup,
+  menu, nav, output, ruby, section, summary,
+  time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+  }
+
+  h2, h4, h5 {
+    font-weight: 700;
+  }
+
+  h2{
+    font-size: 20pt;
+  }
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+
+  }
+
+  body{
+    font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    color: #444;
+    background-color: #ddd;
+    font-size:9.5pt;
+    line-height: 1.2;
+  }
+
+  .border-above{
+    border-top:1px solid #bbb;
+    padding-top:8pt;
+    margin-bottom: 15pt;
+  }
+
+  .border-around{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space